apps/rush-lib/src/index.ts

```diff
@@ -45,7 +45,14 @@ export { EventHooks, Event } from './api/EventHooks';
 
 export { ChangeManager } from './api/ChangeManager';
 
-export { LastInstallFlag as _LastInstallFlag } from './api/LastInstallFlag';
+export { FlagFileBase as _FlagFileBase } from './api/FlagFileBase';
+
+export {
+  ILastInstallFlagJson as _ILastInstallFlagJson,
+  LastInstallFlag as _LastInstallFlag
+} from './api/LastInstallFlag';
+
+export { ILastLinkFlagJson as _ILastLinkFlagJson, LastLinkFlag as _LastLinkFlag } from './api/LastLinkFlag';
```

Comment:
Do any of these actually need to be exposed?

Reply:
Probably not! I can remove them all

Reply:
I've removed them all. This does mean that we won't be exporting `LastInstallFlag` anymore (since we'd also want to export the base class and interface along with it) but let me know if those should be included

Reply:
I added back `LastInstallFlag` as this is used by `@microsoft/rush`, but I've left the `LastLinkFlag` out